src/playlist-loader.js

```diff
@@ -9,6 +9,7 @@ import resolveUrl from './resolve-url';
 import {mergeOptions} from 'video.js';
 import Stream from './stream';
 import m3u8 from 'm3u8-parser';
+import {window} from 'global';
```

Comment:
No strong preference, `import window from 'global/window'` is nice because that's what we use elsewhere.